projects/plugins/jetpack/views/admin/network-settings.php

```diff
@@ -1,86 +1,67 @@
-<?php extract( $data ); ?>
+<?php
+/**
+ * Jetpack Network Settings view template.
+ *
+ * @package automattic/jetpack
+ */
 
-<?php if ( isset( $_GET['updated'] ) && 'true' == $_GET['updated'] ) : ?>
+if ( isset( $_GET['updated'] ) && 'true' === $_GET['updated'] ) : // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	?>
 	<div class="updated"><p><?php esc_html_e( 'Jetpack Network Settings Updated!', 'jetpack' ); ?></p></div>
 <?php endif; ?>
 
-<?php if ( isset( $_GET['error'] ) && 'jetpack_protect_whitelist' == $_GET['error'] ) : ?>
+<?php
+if ( isset( $_GET['error'] ) && 'jetpack_protect_whitelist' === $_GET['error'] ) : // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	?>
 	<div class="error"><p><?php esc_html_e( 'One of your IP addresses was not valid.', 'jetpack' ); ?></p></div>
 <?php endif; ?>
 
 <div class="wrap">
-	<h2><?php _e( 'Network Settings', 'jetpack' ); ?></h2>
+	<h2><?php esc_html_e( 'Network Settings', 'jetpack' ); ?></h2>
 	<form action="edit.php?action=jetpack-network-settings" method="POST">
-		<h3><?php _ex( 'Global', 'Affects all sites in a Multisite network.', 'jetpack' ); ?></h3>
-		<p><?php _e( 'These settings affect all sites on the network.', 'jetpack' ); ?></p>
+		<h3><?php echo esc_html_x( 'Global', 'Affects all sites in a Multisite network.', 'jetpack' ); ?></h3>
+		<p><?php esc_html_e( 'These settings affect all sites on the network.', 'jetpack' ); ?></p>
 		<?php wp_nonce_field( 'jetpack-network-settings' ); ?>
 		<table class="form-table">
-<?php /*
 			<tr valign="top">
-				<th scope="row"><label for="auto-connect">Auto-Connect New Sites</label></th>
+				<th scope="row"><label for="sub-site-override"><?php esc_html_e( 'Sub-site override', 'jetpack' ); ?></label></th>
 				<td>
-					<input type="checkbox" name="auto-connect" id="auto-connect" value="1" <?php checked($options['auto-connect']); ?> />
-					<label for="auto-connect">Automagically connect all new sites in the network.</label>
-				</td>
-			</tr>
-/**/ ?>
-			<tr valign="top">
-				<th scope="row"><label for="sub-site-override"><?php _e( 'Sub-site override', 'jetpack' ); ?></label></th>
-				<td>
-					<input type="checkbox" name="sub-site-connection-override" id="sub-site-override" value="1" <?php checked($options['sub-site-connection-override']); ?> />
-					<label for="sub-site-override"><?php _e( 'Allow individual site administrators to manage their own connections (connect and disconnect) to <a href="//wordpress.com">WordPress.com</a>', 'jetpack' ); ?></label>
+					<input type="checkbox" name="sub-site-connection-override" id="sub-site-override" value="1" <?php checked( $data['options']['sub-site-connection-override'] ); // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable ?> />
```

Comment:
[feel free to ignore] If you want you could do like I did in the Beta Plugin to have phpcs see `$data` as being defined. e.g.

https://github.com/Automattic/jetpack/blob/d48ce820e7cf85bbd5d314bfeec4fb116e9c1aac/projects/plugins/beta/src/admin/branch-card.template.php#L15-L20

Reply:
That's a nice little trick. I tend to prefer explicitly ignoring the rule, so it's easier to find later on if one day we're looking for places where the rule is bypassed.

Reply:
There's still the rule ignore there on line 20. 🤷